### DISCUSSION TITLE
Extensions are not getting installed in vscode container

### DISCUSSION THREAD

**Discussion by ishaRenavikar** [golang/vscode-go#2013] [Help]
Hello,

vscode version : 
Version: 1.62.3
Commit: ccbaa2d27e38e5afa3e5c21c1c7bef4657064247
Date: 2021-11-17T08:00:36.721Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.11.0-46-generic

I have a docker container running on my ubuntu host.
I attach to this container from vscode using remote-container extension.
Once the container is attached, I try to install the go extension inside the container.
go extension displays a button - "Install in container <container name>"
The following message is displayed with exclamation:
This extension is disabled in this workspace because it is defined to run in the Remote Extension Host. Please install the extension in 'Container <container name>' to enable

On clicking the button , nothing happens.
I dont understand why any extension is not getting installed in vscode container.

Please help me.

**Comment by hyangah** [comment]
Have you reported the issue to https://github.com/microsoft/vscode-remote-release/issues ?
This can be confusion from UI but I may be wrong - the extension should be installed on the remote side, and the extension info installed locally will be shown with the mentioned warning message when vscode is attached to the container.

If successfully installed, the extension should be shown in the extension view with the remote container icon in the remote extension side.
![Screen Shot 2022-01-13 at 6 20 33 PM](https://user-images.githubusercontent.com/4999471/149425342-07d1421c-a793-4d6a-8bff-929693ba2568.png)
